src/ch02-02-compile-deploy-interact.md

```diff
@@ -66,14 +66,20 @@ is a smart contract with a private key for signing transactions, while
 the Account Descriptor is a JSON file detailing the wallet’s address and
 public key.
 
-1.  Use Braavos or Argent X browser extensions to create your smart
-    wallet.
+For an account to be used as a signer it must be deployed to the appropriate network (Starknet Goerli or mainnet) and funded.
 
-2.  Follow the provided instructions for your chosen wallet (Argent or
-    Braavos).
+Steps for deploying and funding an account: 
 
-3.  Fund your wallet with ETH. Use [Starknet Goerli
-    Faucet](https://faucet.goerli.starknet.io/) as needed.
+1. Install Braavos or Argent X browser extensions.
+
+2. Fund your account with ETH. Click on "Deposit" on Braavos or "Add funds" on Argent X, then select 
+    "Starknet token faucet" to get testnet tokens. You can also use [Starknet Goerli
+    Faucet](https://faucet.goerli.starknet.io/).
+
+3. For Braavos, on the account page click on "set up your account on-chain" to deploy your account. 
+    For Argent X, fund your account using the faucet (for testnet) or fund using Ether, then make an 
+    outgoing transaction from the account (you can just send some tokens to account2 in your wallet).
```

Comment:
```suggestion
    outgoing transaction from the account. For example, you can just send some tokens to account2 in your wallet.
```
A general technical writing guideline for parentheses is to avoid them. Usually the parenthetical phrase falls into one of these:
* Important enough for its own sentence.
* Not important, so shouldn't be there.
I use parenthese for labeling, like you did in the first part of the sentence: `(for testnet)`.